content/reference-rippled.md

```diff
@@ -10693,6 +10701,311 @@ The response follows the [standard format](#response-formatting), with a success
 * Any of the [universal error types](#universal-errors).
 * `badSeed` - The request provided an invalid secret value. This usually means that the secret value appears to be a valid string of a different format, such as an account address or validation public key.
 
+## validator_lists ##
+[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/ValidatorLists.cpp "Source")
+
+The `validator_lists` command returns human readable information about the current list of published and trusted validators used by the server. [New in: rippled 0.90.0]
```

Comment:
> `[New in: rippled 0.90.0]`

Please add an extra `[]` to the end there. (Markdown style / compatibility thing)